### PR TITLE
Added braced lists to remaining files

### PR DESCRIPTION
**Pull request by Abilityguy**
Reference Issue #2699
This PR covers all the remaining files under this issue.

### CODE REVIEW

File: src/mlpack/tests/metric_test.cpp [hunk 1]
```diff
@@ -164,7 +164,7 @@ TEST_CASE("NMSMetricTest", "[MetricTest]")
   // Selected indices of bounding boxes using
   // torchvision.ops.nms().
   desiredIndices = arma::ucolvec(2);
-  desiredIndices << 0 << 2;
+  desiredIndices = { 0, 2 };
```
Comment (zoq) [inline src/mlpack/tests/metric_test.cpp:167]: Looks like we can remove the line above.

Comment (Abilityguy) [reply src/mlpack/tests/metric_test.cpp:166]: Done.

**Review by zoq — APPROVED** [review]
Thanks for putting this together, no more comments from my side.

**Comment by zoq** [comment]
Thanks again for the contribution.

**Pull request merged**